package.json

```diff
@@ -61,7 +60,6 @@
     "lint-staged": "14.0.1",
     "npm-run-all": "4.1.5",
     "nyc": "15.1.0",
-    "postcss": "8.4.31",
```

Comment:
[Q] Does stylelint definitely not require this? @julieg18 was the one who added it, maybe she remembers

Reply:
I'm not getting any errors or warnings when running `yarn lint:css`. I'll wait on @julieg18 to merge.

Reply:
I don't think stylelint requires it. It should work correctly without it. My guess is I accidentally added it when I was running get started scripts or I added it when I was troubleshooting.